synctl

```diff
@@ -1774,6 +2076,60 @@ class SyntheticTest(Base):
                         output_lists.append(t)
         print('total:', len(output_lists))
 
+
+    def print_synthetic_alerts(self, alerts):
+        id_length = 25
+        max_label_length = 50
+        severity_length = 10
+
+        print(self.fill_space("ID".upper(), id_length),
+              self.fill_space("label".upper(), max_label_length),
+              self.fill_space("severity".upper(), severity_length),
+              "Tests".upper())
+
+        for x in alerts:
+            if len(x["syntheticTestIds"]) > 0:
+                test_str = ','.join(x['syntheticTestIds'])
+            else:
+                test_str = NOT_APPLICABLE
+            print(self.fill_space(x["id"], id_length),
+                  self.fill_space(x["name"], max_label_length),
+                  self.fill_space(str(x["severity"]), severity_length),
+                  self.fill_space(test_str))
+
+    def print_a_alert_details(self, alert_id, single_alert, show_details=False, show_json=False):
+        """Show a smart alert details data"""
+        if single_alert is None or len(single_alert) == 0 or alert_id is None:
+            print("no smart alert")
+            return
+        elif show_json is True:
+            # print alert json
+            print(json.dumps(single_alert[0]))
+        elif show_details is True:
+            self.__print_a_alert_details(single_alert)
+
+    def __print_a_alert_details(self, single_alert):
+        a_single_alert = single_alert[0]
+        print(self.fill_space("Name".upper(), 30), "value".upper())
+        for key, value in a_single_alert.items():
+            if key == 'createdAt' or key == 'modifiedAt' or key == 'observedAt':
```

Comment:
update the key to `created` and `initialCreated`
<img width="444" alt="image" src="https://github.com/instana/synthetic-synctl/assets/25629629/bbeb7724-7ecd-4f47-9e54-dd32ff2c5840">